api/v1/controllers/EventController.js

```diff
@@ -40,7 +41,16 @@ function getAllLocations (req, res, next) {
 function createEvent (req, res, next) {
     services.EventService.createEvent(req.body)
         .then(function (result) {
-            res.body = result.toJSON();
+            result.event = result.event.toJSON();
```

Comment:
There isn't a `.toJSON()` for the result here, which is: 
```
{
    event: EventModel,
    eventLocations: [Locations]
}
```
so we overwrite event with the model's `.toJSON()` and overwrite each Location in the eventLocations array with that models `.toJSON()`, is there a better way of handling this?